packages/std/src/math/decimal.rs

```diff
@@ -30,7 +30,7 @@ impl Decimal {
     const DECIMAL_FRACTIONAL: Uint128 = Uint128::new(1_000_000_000_000_000_000u128); // 1*10**18
     const DECIMAL_FRACTIONAL_SQUARED: Uint128 =
         Uint128::new(1_000_000_000_000_000_000_000_000_000_000_000_000u128); // (1*10**18)**2 = 1*10**36
-    const DECIMAL_PLACES: usize = 18; // This needs to be an even number.
+    pub const DECIMAL_PLACES: u32 = 18; // This needs to be an even number.
 
     pub const MAX: Self = Self(Uint128::MAX);
```

Comment:
I'd group the public ones together and maybe give them a nice doc string. But no blocker.